Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1886,19 +1897,12 @@ CGImageRef CGBitmapContextGetImage(CGContextRef context) {
     return ((__CGBitmapContext*)context)->Impl().image.get();
 }
 
-CGContextRef _CGBitmapContextCreateWithTexture(
-    int width, int height, float scale, DisplayTexture* texture, DisplayTextureLocking* locking) {
-    CGImageRef newImage = nullptr;
-    __CGSurfaceInfo surfaceInfo = _CGSurfaceInfoInit(width, height, _ColorARGB);
-    newImage = new CGGraphicBufferImage(surfaceInfo, texture, locking);
-
-    ComPtr<ID2D1RenderTarget> renderTarget = newImage->Backing()->GetRenderTarget();
-    renderTarget->SetDpi(96 * scale, 96 * scale);
-
+CGContextRef _CGBitmapContextCreateWithTexture(CGImageRef texture, float scale) {
     __CGBitmapContext* context = __CGBitmapContext::CreateInstance(kCFAllocatorDefault);
```

Comment:
Given QuartzCore expects a BitMapContext, we still need the image. 
But the rendering target it moved to QuartzCore and passed here.

---
In reply to: [87046864](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r87046864) [](ancestors = 87046864)